client/src/components/Shared/Form/index.js

```diff
@@ -13,15 +15,95 @@ import {
   StyledLabels,
   StyledInput,
   StyledTextarea,
-  StyledImgCon,
-}
-  from './form.style';
+  StyledImgCon
+} from "./form.style";
 
 class Form extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      isOpen: false,
+      selectedCat: null,
+      selected_itemType: "",
+      selected_colors: "",
+      selected_brands: "",
+      selected_condition: "",
+      selected_labelSize: "",
+      selected_age: "",
+      categories: [
+        "Item Type*",
+        "Color*",
+        "Brand",
+        "Other",
+        "Condition",
+        "Label size",
+        "Age",
+        "purchase price"
+      ],
+      itemType: [
+        "shirt",
```

Comment:
All values need to start with a capital letter. Please change this for the whole PR.

Reply:
but this can be done once we add the real data, this is only for test